sdks/python/pom.xml

```diff
@@ -39,131 +39,165 @@
-                <argument>${python.build.base}</argument>
-              </arguments>
-            </configuration>
-          </execution>
-          <execution>
-            <id>setup-test-create-python-base</id>
-            <phase>process-test-resources</phase>
-            <goals>
-              <goal>exec</goal>
-            </goals>
-            <configuration>
-              <executable>mkdir</executable>
-              <arguments>
-                <argument>-p</argument>
-                <argument>${python.user.base}</argument>
-              </arguments>
-            </configuration>
-          </execution>
-          <execution>
-            <id>setup-test-tox</id>
-            <phase>process-test-resources</phase>
-            <goals>
-              <goal>exec</goal>
-            </goals>
-            <configuration>
-              <executable>${python.pip.bin}</executable>
-              <arguments>
-                <argument>install</argument>
-                <argument>--user</argument>
-                <argument>--upgrade</argument>
-                <argument>--ignore-installed</argument>
-                <argument>tox</argument>
-              </arguments>
-              <environmentVariables>
-                <PYTHONUSERBASE>${python.user.base}</PYTHONUSERBASE>
-              </environmentVariables>
-            </configuration>
-          </execution>
-          <execution>
-            <id>setuptools-test</id>
-            <phase>test</phase>
-            <goals>
-              <goal>exec</goal>
-            </goals>
-            <configuration>
-              <executable>${python.user.base}/bin/tox</executable>
-              <arguments>
-                <argument>-e</argument>
-                <argument>py27</argument>
-                <argument>-c</argument>
-                <argument>tox.ini</argument>
-              </arguments>
-              <environmentVariables>
-                <PYTHONUSERBASE>${python.user.base}</PYTHONUSERBASE>
-              </environmentVariables>
-            </configuration>
-          </execution>
-          <execution>
-            <id>setuptools-sdist</id>
-            <phase>package</phase>
-            <goals>
-              <goal>exec</goal>
-            </goals>
-            <configuration>
-              <executable>${python.interpreter.bin}</executable>
-              <arguments>
-                <argument>setup.py</argument>
-                <argument>sdist</argument>
-                <argument>--dist-dir</argument>
-                <argument>${project.build.directory}</argument>
-              </arguments>
-              <environmentVariables>
-                <PYTHONUSERBASE>${python.user.base}</PYTHONUSERBASE>
-              </environmentVariables>
-            </configuration>
-          </execution>
-        </executions>
-      </plugin>
-    </plugins>
-  </build>
+  <profiles>
+    <profile>
+      <id>beam-sdks-python-package</id>
```

Comment:
Activation based on OS can only be set in a profile, I'm not very familiar with maven but was following

[1] https://github.com/apache/beam/blob/master/sdks/java/maven-archetypes/pom.xml#L60
[2] http://stackoverflow.com/questions/8963713/maven-condition-based-on-os-family